tests/integration/records/single_db/test_records_unload_datetime.py

```diff
@@ -0,0 +1,353 @@
+                                     create_sample(datetimeformattz).replace('-00', '.000000+0000\n'),
+                                     # TODO: Should this be necessary?
+                                     create_sample(datetimeformattz) +
+                                     f":{SAMPLE_SECOND:02d}.000000\n"
+                                     ],
+                          f"from datetimeformattz {datetimeformattz} and addl_hints {addl_hints}")
+
+    def test_unload_timeonly(self) -> None:
+        self.createTimeTable()
+        matching_dateformat = {
+#            'YYYY-MM-DD HH:MI:SS': 'YYYY-MM-DD',
+#            'YYYY-MM-DD HH12:MI AM': 'YYYY-MM-DD',
+#            'MM/DD/YY HH24:MI': 'MM/DD/YY',
+#            'YYYY-MM-DD HH24:MI:SSOF': 'YYYY-MM-DD',
+        }
+        for timeonlyformat in TIMEONLY_CASES:
+            addl_hints: PartialRecordsHints = {}
+            # if self.engine.name == 'redshift':
+            #     if datetimeformattz != 'YYYY-MM-DD HH:MI:SSOF':
+            #         # this is the only format supported by Redshift on
+            #         # export, so we're going to need to be sure to use
+            #         # hints that work in Pandas - i.e.,
+            #         addl_hints = {
+            #             'dateformat': matching_dateformat[datetimeformattz],
+            #             'datetimeformat': datetimeformattz.replace('OF', ''),
+            #         }
+            #         if 'AM' in datetimeformattz:
+            #             # TODO: Add a GitHub issue for this
+            #             logger.warning('Cannot export this dateformattz using Pandas or Redshift--'
+            #                            'skipping test')
+            #             continue
+            records_format = RecordsFormat(variant=VARIANT_FOR_DB[self.engine.name],
+                                           hints={
+                                               'timeonlyformat': timeonlyformat,
+                                               'compression': None,
+                                               'header-row': False,
+                                               **addl_hints,  # type: ignore
+                                           })
+            csv_text = self.unload(column_name='time',
+                                   records_format=records_format)
+            allowed_items = [create_sample(timeonlyformat) + "\n",
+                                     # create_sample(datetimeformattz).replace('-00', '+00') + "\n",
+                                     # # TODO: Should this be necessary?
+                                     # create_sample(datetimeformattz) + ".000000\n",
+                                     # # TODO: Should this be necessary?
+                                     # create_sample(datetimeformattz).replace('-00', '.000000+0000\n'),
+                                     # # TODO: Should this be necessary?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker